src/containers/PatientProfilePage.tsx

```diff
@@ -228,12 +229,15 @@ const PatientProfilePage = ({
             }}
           />
           <RadioSelector
-            labels={['Male', 'Female']}
+            labels={Object.keys(Gender)}
             currentValue={formFields.gender}
             handleChange={(e: React.ChangeEvent<HTMLElement>) => {
               setFormFields({
                 ...formFields,
-                gender: (e.target as HTMLInputElement).value,
+                gender:
+                  (e.target as HTMLInputElement).value === Gender.M
+                    ? Gender.M
+                    : Gender.F,
```

Comment:
how about this syntax instead? it's just very ugly as it is currently lol
```suggestion
                  (e.target as HTMLInputElement).value as Gender
```